content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 90
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates used by the webhooks of Galley and the sidecar
+injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+This task shows how to provision and manage DNS certificates in Istio through
+Chiron: a lightweight component that signs certificates
+using Kubernetes CA APIs without maintaining its own private key.
+
+## Before you begin
+
+* Install Istio through `istioctl` with DNS certificates configured.
```

Comment:
Does enabling this feature require a reinstall of Istio? Perhaps a note would be appropriate then?

Reply:
Done: added a note.